v2/mongodb-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/mongodb/options/MongoDbToBigQueryOptions.java

```diff
@@ -94,4 +95,25 @@ public interface BigQueryWriteOptions extends PipelineOptions, DataflowPipelineO
 
     void setOutputTableSpec(String outputTableSpec);
   }
+
+  /** UDF options. */
+  public interface JavascriptDocumentTransformerOptions extends PipelineOptions {
+    @TemplateParameter.Text(
+        order = 3,
+        description = "Path to the UDF stored in the GCS bucket.",
+        helpText = "Enter the gcs path in format gs://<bucket-name>/<js-file>.js .",
+        example = "gs://test-bucket/test.js")
+    String getJavascriptDocumentTransformGcsPath();
+
+    void setJavascriptDocumentTransformGcsPath(String javascriptDocumentTransformGcsPath);
+
+    @TemplateParameter.Text(
```

Comment:
`optional = true` is missing from the annotation.

Reply:
updated